### ISSUE TITLE
Unicode decode error

### ISSUE THREAD

**Issue by tse-tamir** [ansible/ansible#15644]
I got the following error when I execute ansible command from terminal. Can you please help me to solve this problem?

version: ansible 2.0.2.0

![untitled](https://cloud.githubusercontent.com/assets/8093401/14884897/53968f3a-0d48-11e6-96bd-9160b69506a6.png)

##### ISSUE TYPE

- Bug Report
- Feature Idea
- Documentation Report
##### ANSIBLE VERSION

```
<!--- Paste verbatim output from “ansible --version” between quotes -->
```
##### STEPS TO REPRODUCE

```
<!--- Paste example playbooks or commands between quotes -->
```

##### ACTUAL RESULTS

```
<!--- Paste verbatim command output between quotes -->
```

**Comment by nitzmahone** [comment]
@tse-tamir it looks like you might be running the playbook from a directory with non-ASCII characters (homedir with accented chars in username?). Just to help us narrow down the issue, can you try copying your playbooks to /tmp or another dir whose path is just straight ASCII, and see if you have the same behavior?

**Comment by tse-tamir** [comment]
Thanks! It works!

**Comment by nitzmahone** [comment]
@tse-tamir Great! We'd like to figure out the root cause of the issue though so that we might be able to reproduce the issue and fix it- can you share your OS version, your username, and what OS language you're configured for?

Thanks!

**Comment by abadger** [comment]
Simple reproducer: 

```
$ mkdir café
$ cd café
$ echo > 15644.yml  << EOF
---
- hosts: localhost
  tasks:
  - ping:
EOF
$ LC_ALL=C ansible-playbook 15644.yml
```

Now fixed with commit   b8a988e in both devel and stable-2.1
